v1/nix/modules/drvs/ansible/lock-x86_64-linux.json

```diff
@@ -1,5 +1,8 @@
 {
   "mach-nix": {
-    "pythonSources": "sha256-dCo1llHcCiFrBOEd6mWhwqwVglsN2grSbcdBj8OzKDY="
+    "pythonSources": {
+      "hash": "sha256-8qYhbRbR7NBvJcp8zKCG3yT8IWLfyyOa/2mFPoRt4yk=",
+      "maxDate": "2023-03-16"
+    }
```

Comment:
I wouldn't worry about the exact structure of this too much yet.
Eventually we should have fetchers as modules, and it should be the responsibility of the fetcher to make use of the lock file, not the mach-nix module.